<launch>
	<node name="serial_node" pkg="rosserial_python" type="serial_node.py" output="screen"> 	 <!-- Node to begin rosserial -->
		<param name="port"     type="string" value="/dev/ttyUSB0"/>
		<param name="baud"     type="int"    value="115200"/>
	</node>
	<node name="serial_node_arduino" pkg="rosserial_python" type="serial_node.py" output="screen"> 	 <!-- Node to begin rosserial -->
		<param name="port"     type="string" value="/dev/ttyACM0"/>
		<param name="baud"     type="int"    value="57600"/>
	</node>
</launch>
	
